Drop-In/src/test/java/com/braintreepayments/api/dropin/DropInActivityUnitTest.java

```diff
@@ -527,6 +536,27 @@ public void anyExceptionExitsActivityWithError() {
         assertExceptionIsReturned("sdk-error", new Exception("Error!"));
     }
 
+    @Test
+    public void testAndroidPayVisible() {
```

Comment:
I think this test can be removed and a test be written in `DropInActivityPowerMockTest`. PowerMock gives us the ability to mock test on our static payment method classes.

Try a test like this in `DropInActivityPowerMockTest`
```
    @Test
    public void onConfigurationFetch_whenDisabledAndroidPay_doesNotCallAndroidPay() {
        mockStatic(AndroidPay.class);

        DropInRequest dropInRequest = new DropInRequest()
                .disableAndroidPay();

        mActivity.setDropInRequest(dropInRequest);
        mActivity.onConfigurationFetched(null);

        verifyStatic(Mockito.never());
        AndroidPay.isReadyToPay(any(BraintreeFragment.class), any(BraintreeResponseListener.class));
    }
```